src/carbon_alkalinity_nutrients.jl

```diff
@@ -222,16 +222,16 @@ end
 end
 
 """
-    dissolved_organic_phosphate_remin(remineralization_rate, 
+    dissolved_organic_phosphorus_remin(remineralization_rate, 
                                       dissolved_organic_phosphorus_concentration)
-Calculate the remineralization of dissolved organic phosphate.
+Calculate the remineralization of dissolved organic phosphorus.
 """
-@inline dissolved_organic_phosphate_remin(remineralization_rate, 
+@inline dissolved_organic_phosphorus_remin(remineralization_rate, 
                                           dissolved_organic_phosphorus_concentration) = 
         max(0, remineralization_rate * dissolved_organic_phosphorus_concentration)
 
 # Martin Curve
-@inline particulate_organic_phosphate_remin() = 0.0
+@inline particulate_organic_phosphorus_remin() = 0.0
```

Comment:
```suggestion
@inline particulate_organic_phosphorus_remin() = 0
```

doesn't matter at all, but its good just to build a habit of not hardcoding Float64

Reply:
Got it